evm/interchainTokenFactory.js

```diff
@@ -228,21 +238,30 @@ async function processCommand(_axelar, chain, chains, options) {
         }
 
         case 'deployRemoteCanonicalInterchainToken': {
-            const { tokenAddress, destinationChain, gasValue } = options;
+            const { tokenAddress, destinationChain, gasValue, env } = options;
+
+            const submittedGasValue = !gasValue
+                ? await calculateItsCrossChainGas({
+                      sourceChain: chain.axelarId,
+                      destinationChain,
+                      env,
+                      eventType: 'InterchainTokenDeployment',
```

Comment:
`options` as an arg is meant for pass through from CLI args. If you're just creating your own object, it's worse since you lose the type structure. So I'd pass in as separate args